docs/upgrade-guide.md

```diff
@@ -1,5 +1,12 @@
 # Upgrade Guide
 
+## Upgrading from deck.gl v8.8 to v8.9
+
+#### Breaking changes
+
+- `TextLayer`'s `maxWidth` is now relative to the font size. This change aims to make this prop more intuitive to use. If you have been using this prop in previous versions, divide its value by `64` (or `fontSettings.fontSize`).
```

Comment:
Isn't it relative to pixel size now, and was _previously_ font size?

Reply:
If we do choose to break the prop, should we print a warning in console? I remember a lot of people being affected by the size changes we did a while back.

Reply:
Added a warning